### PR TITLE
Add plans for redis by VSHN

### PR DESCRIPTION
**Pull request by glrf**
This PR adds plans for Redis by VSHN. It's very similar to https://github.com/vshn/component-appcat/pull/114 

PR for end-user documentation: https://github.com/appuio/appcat-docs/pull/45

## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation. *Don't forget to generate the CRD API!*
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [x] Link this PR to related issues or PRs.

### CODE REVIEW

File: class/defaults.yml [hunk 1]
```diff
@@ -133,6 +133,34 @@ parameters:
...
+            standard-4:
+              size:
+                cpu: "1"
+                memory: "4Gi"
+                disk: 16Gi
+            standard-8:
+              size:
+                cpu: "2"
+                memory: "8Gi"
+                disk: 16Gi
```
Comment (glrf) [inline class/defaults.yml:162]: We're using the same disk size for all of them as disk resizing actually doesn't work yet..

File: docs/modules/ROOT/pages/references/services-vshn.adoc [hunk 2]
```diff
@@ -129,3 +129,44 @@ type:: bool
...
+      memory: "4Gi"
+      disk: 16Gi
+  standard-8:
+    enabled: false
+  plus-4:
+    size: ${appcat:services:vshn:redis:plans:standard-4:size}
+    scheduling:
+      nodeSelector:
+        appuio.io/node-class: "plus"
+    note: "Will be scheduled on APPUiO Cloud plus nodes"
```
Comment (wejdross) [inline docs/modules/ROOT/pages/references/services-vshn.adoc:165]: I'd add https://docs.appuio.cloud/user/references/node-classes.html this link somewhere here, so it's easy and fast to find specifics of nodes, not everyone must be aware what "plus" means

**Review by wejdross — APPROVED** [review]

File: component/common.libsonnet [hunk 3]
```diff
@@ -86,6 +86,18 @@ local mergeArgs(args, additional) =
   [ '%s=%s' % [ k, final[k] ] for k in std.objectFields(final) ];
 
 
+local filterDisabledParams(params) = std.foldl(
+  function(ps, key)
+    local p = params[key];
+    local enabled = p != null && p != {} && std.get(p, 'enabled', true);
```
Comment (zugao) [inline component/common.libsonnet:92]: Remember to add enabled in the defaults as well like you did for postgres.

**Review by zugao — APPROVED** [review]
LGTM

**Pull request merged**